fed_test_utils/fed.go

```diff
@@ -153,51 +154,89 @@ func NewFedTest(t *testing.T, originConfig string) (ft *FedTest) {
 	directorStartTime := time.Now().Add(-6 * time.Minute)
 	director.SetStartupTime(directorStartTime)
 
-	config.InitConfig()
 	err = config.InitServer(ctx, modules)
 	require.NoError(t, err)
 
 	// Read in any config we may have set
+	var originConf interface{}
 	if originConfig != "" {
```

Comment:
This check shouldn't be needed anymore because a recent change added:
```
if originConfig == "" {
		originConfig = fedTestDefaultConfig
	}
```
to the start of the test